server/trillian_log_server/main.go

```diff
@@ -17,15 +17,23 @@
 package main
 
 import (
+	"context"
 	"flag"
 	_ "net/http/pprof"
 
 	_ "github.com/go-sql-driver/mysql" // Load MySQL driver
 
+	// Register key ProtoHandlers
+	_ "github.com/google/trillian/crypto/keys/der/proto"
```

Comment:
Suggestion: I've been splitting imports in the following blocks: golang, external libs, aliases and _ imports. I've found it more readable than mixing aliases and _ imports on the "external" block.

Eg:
```go
import (
    "context"
    "flag"

    "github.com/golang/glog"
    "github.com/golang/protobuf/proto"
    "github.com/google/trillian"
    "github.com/google/trillian/cmd"

    mysqlq "github.com/google/trillian/quota/mysql"

    _ "github.com/google/trillian/merkle/objhasher" // Load hashers
    _ "github.com/google/trillian/merkle/rfc6962"   // Load hashers
)
```